doc/en/doctest.rst

```diff
@@ -130,3 +130,20 @@ which can then be used in your doctests directly::
         10
         """
         pass
+
+
+Output format
+-------------
+
+You can change the diff output format on failure for your doctests
+by using one of standard doctest modules format in options
+(see :data:`python:doctest.REPORT_UDIFF`, :data:`python:doctest.REPORT_CDIFF`,
```

Comment:
I did, indeed. Never used intersphinx before so I had to find the syntax first too ^^